cmd/root.go

```diff
@@ -25,6 +25,10 @@ var rootCmd = &cobra.Command{
 			return !val
 		}
 		settings.ShowHeader = oppositeBool("no-header")
+		if !settings.ShowHeader {
+			options.HeaderFromFile = ""
```

Comment:
I don't like this approach, I mean we need to treat everything in `Settings` and `Options` as immutable so instead of changing the value of `HeaderFromFile` on the fly based on some condition I'd suggest to create a new property in `module.Options`, something like `ShowHeader bool` with default to `true` and just set `options.ShowHeader = settings.ShowHeader` instead of these lines.